worlds/timespinner/Regions.py

```diff
@@ -213,6 +213,17 @@ def create_location(player: int, location_data: LocationData, region: Region) ->
     return location
 
 
+def create_regions(world: MultiWorld, player: int, locations_per_region: Dict[str, List[LocationData]],
+                    region_names: List[str]) -> Dict[str, Region]:
+    
+    regions: Dict[str, Region] = {}
+
+    for name in region_names:
+        regions[name] = create_region(world, player, locations_per_region, name)
```

Comment:
`locations_per_region` seems to be unused?

Reply:
its used, its passed on into the create_region method that will create the locations that belong to that region inside it